values.yaml

```diff
@@ -630,6 +630,11 @@ server:
     #   ports:
     #   - protocol: TCP
     #     port: 443
+    podSelector: {}
+    # Restrict traffic to vault pods only with given labels
+    # podSelector:
+    #   matchLabels:
+    #     vault-access: "true"
```

Comment:
I agree with you and just implemented this change.

I did not change anything at the netPol of the injector, because it is not referenced in the values anyway.

https://github.com/hashicorp/vault-helm/blob/9a16496e869abd8b5c6d63ee5119427e9cf2d353/templates/injector-network-policy.yaml#L22-L27